src/pages/guides/getting_started/code_playground_addon_mode.md

```diff
@@ -0,0 +1,245 @@
+  width: 100%;
+  padding: 8px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  margin-top: 10px;
+}
+```
+
+### 3. Iframe JS Tab
+
+This tab is for writing JavaScript code that runs in the iframe context of your add-on. Here, you can interact with:
+
+- The [Add-on UI SDK (`addOnUISdk`)](../../references/addonsdk/index.md)
+- The DOM elements in your HTML
+- Event handlers for your UI components
+
+This environment corresponds to the code you would typically write in your `index.js` or UI JavaScript files in a full add-on project.
+
+**Example:**
+
+```js
+// Wait for the add-on UI SDK to be ready
+addOnUISdk.ready.then(() => {
+  console.log('Add-on UI SDK is ready');
+  
+  // Get references to DOM elements
+  const button = document.getElementById('myButton');
+  const input = document.getElementById('myInput');
+  
+  // Add event listeners
+  button.addEventListener('click', () => {
+    const text = input.value;
+    if (text) {
+      // Send message to Document JS tab
+      addOnUISdk.app.documentSandbox.dispatch({
+        type: 'ADD_TEXT',
+        payload: { text }
+      });
+    }
+  });
+});
+```
+
+### 4. Document JS Tab
+
+This tab is where you write JavaScript code that interacts directly with the Adobe Express document. It runs in the [Document Sandbox](../../references/document-sandbox/index.md) environment and gives you access to:
+
+- Document manipulation capabilities with the [Document APIs](../../references/document-sandbox/document-apis/index.md)
+- [Communication APIs](../../references/document-sandbox/communication/index.md) to facilitate interaction between the iframe context and the Document Sandbox
+
+The Document JS tab corresponds to the code typically found in the `code.js` file of a complete add-on project.
+
+**Example:**
+
+```js
+// Import necessary modules
+import { editor, text } from 'express-document-sdk';
+
+// Listen for messages from the iframe
+addOnUISdk.app.documentSandbox.addEventListener('message', (event) => {
+  if (event.data.type === 'ADD_TEXT') {
+    addTextToDocument(event.data.payload.text);
+  }
+});
+
+async function addTextToDocument(textContent) {
+  try {
+    // Create a new text node
+    const textNode = await text.create({
+      text: textContent,
+      fontSize: 24,
+      fontFamily: 'Arial'
+    });
+    
+    // Add it to the document
+    editor.context.insertNode(textNode);
+    
+    console.log('Text added to document');
+  } catch (error) {
+    console.error('Error adding text:', error);
+  }
+}
+```
+
+## Key Differences from Script Mode
+
+| Feature | Script Mode | Add-on Mode |
+|---------|-------------|-------------|
+| **Global Await** | Yes | No - must use async functions |
+| **Automatic Imports** | Yes | No - must import manually |
+| **UI Building** | No | Yes - full HTML/CSS/JS |
+| **Environment** | Document Sandbox only | Both iframe and Document Sandbox |
+| **API Access** | Document APIs only | Document APIs + Add-on UI SDK |
+
+## Best Practices
```

Comment:
I feel these are general add-on development best practices. Adding them to code playground feels a bit off to me. Can we skip this section? Or move it to another page where we are teaching the concepts of addons in general?